### ISSUE TITLE
Admin error adding content object

### ISSUE THREAD

**Issue by cameroncf** [meldsolutions/Meld-Forums#3]
I'm getting the following error after selecting "Plugins > Meld Forums" in the Mura Admin. Looking into it right now, but if you have any ideas I'm all ears.

![screen shot 2013-08-23 at 10 27 46 am](https://f.cloud.github.com/assets/789760/1016707/420229ba-0c00-11e3-9028-e2a9f9cef021.png)

**Comment by meldsolutions** [comment]
What version of Mura is this? Also, is this an update from an existing MF install or a fresh install?

**Comment by cameroncf** [comment]
There was a previous version of Meld Forums, but I nuked it and the entry in tplugins. I also nuked all mf_ tables before reinstalling. I had a few other issues with it not respecting the table prefix during instal as well, which I attributed to the fact that something was in there before.

I may have borked things up I suppose. I've done a Meld Plugin update and also an application refresh. I'm about to start looking at the plugin config values in the DB next. I'm not really sure what drives this part of the Mura admin.

Mura Core is 6.0.5340 and Site is 6.0.5305.
